fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/identifiers/HierarchyConverter.java

```diff
@@ -162,4 +162,9 @@ public void setPrefix(final String p) {
         this.prefix = p;
     }
 
+    @Override
+    public int hashCode() {
```

Comment:
Not supporting `hashCode()` seems like an a sign of a more fundamental issue.
But any ways, the related Sonar issue seems to be gone: http://sonar.fcrepo.org/drilldown/issues/1?&rule=findbugs%3AEQ_COMPARETO_USE_OBJECT_EQUALS&rule_sev=MAJOR&severity=MAJOR